src/components/players/Audio/Audio.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+import styled, { withTheme } from "styled-components";
+import { Text } from "../../atoms/Typography/Text";
+
+interface TrackProps {
+  src: string;
+  kind: string;
+  srcLang: string;
+  label: string;
+}
+interface AudioPlayerProps {
+  url?: string;
+  onFinish: () => void;
+  track?: TrackProps;
+}
+
+const StyledWrapper = styled("div")``;
+
+export const AudioPlayer: React.FC<AudioPlayerProps> = ({
+  url,
+  onFinish,
+  track,
+}): React.ReactElement => {
+  if (!url) {
+    return (
+      <Text size="14" noMargin>
+        no source url provided
+      </Text>
+    );
+  }
+
+  return (
+    <StyledWrapper>
+      {url && ( // eslint-disable-next-line jsx-a11y/media-has-caption
+        <audio
```

Comment:
don't use browser audio use VideoPlayer component of something similar 

it should look like videoplayer with same capaliticies